docs/desktop-wpf/fundamentals/styles-templates-create-apply-template.md

```diff
@@ -0,0 +1,183 @@
+---
+title: Create a template in WPF - .NET Desktop
+description: Learn how to create and reference a control template in Windows Presentation Foundation and .NET Core.
+author: thraka
+ms.author: adegeo
+ms.date: 10/20/2019
+dev_langs: ["csharp", "vb"]
+helpviewer_keywords: 
+  - "control contract [WPF]"
+  - "controls [WPF], visual structure changes"
+  - "ControlTemplate [WPF], customizing for existing controls"
+  - "skinning controls [WPF]"
+  - "controls [WPF], appearance specified by state"
+  - "templates [WPF], custom for existing controls"
+ms.assetid: 678dd116-43a2-4b8c-82b5-6b826f126e31
+---
+
+# Create a template for a control
+
+With Windows Presentation Foundation (WPF), you can customize an existing control's visual structure and behavior with your own reusable template. Templates can be applied globally to your application, windows and pages, or directly to controls. Most scenarios that require you to create a new control can be covered by instead creating a new template for an existing control.
```

Comment:
```suggestion
With Windows Presentation Foundation (WPF), you can customize an existing control's visual structure and behavior with your own reusable template. Templates can be applied globally to your application, windows, and pages, or directly to controls. Most scenarios that seemingly require creating a new control can be covered by instead creating a new template for an existing control.
```

Reply:
Would it be more appropriate to keep these as a single "thing" in the list of things? In the context of objects, **App** vs **Windows/Pages** vs **Controls** not **App** vs **Windows** vs **Pages** vs **Controls**

Reply:
I read it as apply:

globally - to app, windows, and pages
directly - to controls

Which is apparently now how I was supposed to read it. So how about:

...globally to your application, controls, or windows and pages.